cluster/ubuntu/util.sh

```diff
@@ -364,6 +369,11 @@ function provision-master() {
     ubuntu/binaries/master/ \
     "${MASTER}:~/kube"
 
+  # copy the flannel binaries and scripts, if flannel is being used
```

Comment:
Look at how the `cluster/ubuntu/{master,minion}/init*/flannel*` files have been moved to new locations under `cluster/ubuntu/{master,minion}-flannel/init*/`.  The old locations were among those covered by the copies of `cluster/ubuntu/{master,minion}` (e.g., one is at old line 363, new line 368).